pandas/tseries/offsets.py

```diff
@@ -710,6 +689,7 @@ def __init__(self, **kwds):
         kwds['end'] = self._validate_time(kwds.get('end', '17:00'))
         self.kwds = kwds
         self.offset = kwds.get('offset', timedelta(0))
+        self._offset = self.offset  # alias for backward compat
```

Comment:
why don't you just define ``.offset`` as a property to return ``._offset``?

Reply:
I definitely tried this, don't remember off the top why it didn't work.  Maybe when the caffeine kicks in.

Reply:
ok, let's try that type of refactor again.